qemu.mk

```diff
@@ -41,13 +84,8 @@ define bios-qemu-common
 endef
 
 bios-qemu: buildroot optee-os linux
```

Comment:
The dependencies aren't needed any longer

Reply:
ok for dropping dep on optee-os.
buildroot and linux are still booted by bios but i think both beps can be discarded at build time. can you confirm?

Reply:
Agree. There's other targets that depend on linux and buildroot